### ISSUE TITLE
Angular material template does not work on production build

### ISSUE THREAD

**Issue by robherman** [ngx-rocket/generator-ngx-rocket#205]
## I'm submitting a...
[X] Bug report
[ ] Feature request
[ ] Documentation issue or request

## Current behavior
When I do a production build using the command "npm run build", and deploy my "dist" folder, the following issues occur when serving:

- Login button does not activate when filling username and password.
- md-switch does not toggle off.
- md-select does not work.

So, i guess the minification broke something, but i don't know what might be happenning.

Running on localhost:4200 using npm start works just fine.

## Expected behavior
The production build should work as the local server run.

## Minimal reproduction of the problem with instructions
1) Download generator-ngx-rocket
2) Execute "ngx new" command
3) Select angular-material, responsive header, and authentication.

## Environment
Deployed on IIS 8.5

```
- generator version: latest
- node version: 9.2.1
- npm version: 5.5.1
- OS:  Building on MacOS High Sierra. Deployed on Windows Server 2012.
```

**Comment by sinedied** [comment]
Thanks for the report! Could you try if it work when using `angular-cli@1.5`?
```
npm i angular-cli@1.5
```
I suspect a regression in latest `angular-cli@1.6` versions, and that would not be the first 😉 

You can also try with `npm start --prod` to run the app on the local server using the production build (without service worker).

I'll also investigate on my end if something happened during the angular-maaterial@5 final version switch.

**Comment by southapps** [comment]
Thanks for the quick response @sinedied ! 
Well, it works when running npm start --prod. But issues still exist using npm run build, and serving the generated dist folder (used http-server).
Same behaviour using angular-cli@1.5.3 or angular-cli@1.6.2.

**Comment by sinedied** [comment]
Thanks for the feedback.
I'm out of office for now but I'll check this issue ASAP.
An updated branch for 1.6 is ready but I'm waiting a new angular-cli release to fix a bug with SCSS that was introduced recently before merging, that should not take too long as the fix is already merged 😉

**Comment by southapps** [comment]
@sinedied Cool! Thanks.

**Comment by sinedied** [comment]
Fixed by #224
